tests/Fixer/CastNotation/ModernizeTypesCastingFixerTest.php

```diff
@@ -188,11 +188,18 @@ public function provideFix70Cases()
                 '<?php $foo = ((string) ($x + $y))[0];',
                 '<?php $foo = strval($x + $y)[0];',
             ],
-            [
+        ];
+
+        foreach ($tests as $index => $test) {
+            yield $index => $test;
+        }
+
+        if (\PHP_VERSION_ID < 80000) {
```

Comment:
should separate this out

Reply:
most of these are fine IMHO, is just not worth the effort to do all of these in separated methods. If someone wants to PR out later than that is totally fine, for now I just want the test being green so we can keep going.